apps/payments/next/app/layout.tsx

```diff
@@ -1,5 +1,5 @@
 import ExampleHeader from '../components/ExampleHeader';
-import './global.css';
+import '@fxa/shared/styles';
```

Comment:
Could we revert this change, and instead import the shared styles into global.css?

It's possible that in future there will be styles specific to payments-next that shouldn't be added to the shared styles, and having the global.css at that point would make more sense.

It's also the recommended approach of the [nx implementation docs.](https://nx.dev/recipes/react/using-tailwind-css-in-react#step-3:-import-tailwind-css-styles)

Reply:
Should the styles from fxa-payments-server be placed in this restored global.css as well?

Reply:
That'd be nice, but I think it's also fine if we add them as and when we need them.